ocs_ci/ocs/ui/base_ui.py

```diff
@@ -248,6 +248,16 @@ def get_elements(self, locator):
         """
         return self.driver.find_elements(by=locator[1], value=locator[0])
 
+    def get_element_attribute(self, locator, attribute, safe: bool = False):
+        """
+        Get attribute from WebElement
+        """
+        el = self.get_elements(locator)
+        if safe:
+            if not len(el):
+                return
+        return el[0].get_attribute(attribute)
```

Comment:
Why hard code the index to 0th position when there could be other elements in the list returned by`get_elements`?

Reply:
it's been already checked on the length of the found web_elements list two lines above - the case for the **safe** attribute extraction.
For the **unsafe**(safe=False) we intentionally referring to the element and want test to fail if any element not found by self.get_elements(locator)